packages/coze-js/src/version.ts

```
@@ -0,0 +1,116 @@
+
+  let osName = platform.toLowerCase();
+  let osVersion = os.release();
+
+  if (platform === 'darwin') {
+    osName = 'macos';
+    // Try to parse the macOS version
+    try {
+      const darwinVersion = os.release().split('.');
+      if (darwinVersion.length >= 2) {
+        const majorVersion = parseInt(darwinVersion[0], 10);
+        if (!isNaN(majorVersion) && majorVersion >= 9) {
```

Biome

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)